Changelog - First Release (#120)
aerogear/mobile-security-service-operator: CHANGELOG.md

## Released 
 
## [0.1.0] - Wed Jun 12 2019

> - Initial version meeting requirements defined for the Operator which is managing the [MobileSecurityService](https://github.com/aerogear/mobile-security-service) and its database

Reviewer: ```suggestion
- Initial release of operator that meets requirements to manage [MobileSecurityService](https://github.com/aerogear/mobile-security-service) and its database.
```
